bitnami/nginx/README.md

```diff
@@ -97,6 +97,7 @@ The command removes all the Kubernetes components associated with the chart and
 | Name                                    | Description                                                                               | Value   |
 | --------------------------------------- | ----------------------------------------------------------------------------------------- | ------- |
 | `replicaCount`                          | Number of NGINX replicas to deploy                                                        | `1`     |
+| `strategy`                              | Strategy for the deployment                                                               | `{}`    |
```

Comment:
Please rename this parameter to "updateStrategy". We're using this name on every chart regardless it's a deployment or a statefulset.